holidays/countries/barbados.py

```diff
@@ -0,0 +1,69 @@
+from holidays.calendars.gregorian import JAN, APR, JUL, AUG, NOV, DEC
+from holidays.holiday_base import HolidayBase
+from holidays.holiday_groups import ChristianHolidays, InternationalHolidays
+
+
+class Barbados(HolidayBase, ChristianHolidays, InternationalHolidays):
+    """
+    https://www.timeanddate.com/holidays/barbados/
+    """
+
+    country = "BB"
+
+    def __init__(self, *args, **kwargs) -> None:
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        super().__init__(*args, **kwargs)
+
+    def _populate(self, year):
+        super()._populate(year)
+
+        # Overview: https://en.wikipedia.org/wiki/Public_holidays_in_Barbados
+        # New Year's Day
+        self._add_new_years_day("New Year's Day")
+
+        # Errol Barrow Day
+        self._add_holiday("Errol Barrow Day", JAN, 21)
+
+        # Good Friday
+        self._add_good_friday("Good Friday")
+
+        # Easter Monday
+        self._add_easter_monday("Easter Monday")
+
+        # National Heroes Day
+        self._add_holiday("National Heroes Day", APR, 28)
+
+        # Labor Day / May Day
+        self._add_labor_day("Labor Day / May Day")
```

Comment:
```suggestion
        # May Day
        self._add_labor_day("May Day")
```
According to government sources, e.g. [here](https://gisbarbados.gov.bb/download/public-holidays-for-2023/), it's "May Day".

Reply:
Thanks for looking at it. I am going to be busy the next few days with moving, but I will come back and address all of these :)